### PR TITLE
Add --- Participations relation between student and livequiz

### PR DESCRIPTION
**Pull request by adomasciplys**
# Description
Methods for inserting a participation into the database have been created

## What is added/changed/removed
- Created participation class in models folder
- Added function to add participation to the "participation" table(actually called livequiz_quiz_student in db)
- Added function to get all participations based on livequizid and studentid
- Added a service layer function new_participation which calls the participation db methods
- Tests of insertion and retrieval of a participation from the database
 

## Issue/fixes reference
Number: https://github.com/orgs/AAU-P5-Moodle/projects/2/views/1?pane=issue&itemId=85501729&issue=AAU-P5-Moodle%7Cmoodle-1%7C55

The functions are not called from the 'front-end' exactly. However, we have a way to call any function from the front-end prepared in our branch https://github.com/AAU-P5-Moodle/moodle-1/tree/Individual-Quiz-Runner-Session which will be merged later

## Testing
Just run the phpunit testing suite

# Checks
- [x] Updated / added tests for these changes
- [x] PHPunit locally passed
- [x] Codesniffer locally passed
- [ ] Behat locally passed - (THESE TESTS ARE NOT RELEVANT FOR THE PR)

### CODE REVIEW

**Review by vjohns22 — DISMISSED** [review]
LGTM 👍

**Review by ChristofferGamel — APPROVED** [review]
Looks good! Fedt at i bruger servicelaget, og god dokumentation 👍

File: server/moodle/mod/livequiz/classes/models/student_quiz_relation.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace mod_livequiz\models;
+
+use dml_exception;
+use dml_transaction_exception;
+
+/**
+ * 'Static' class, do not instantiate.
+ * Displays the livequiz view page.
```
[the remark this answers is not in the document]
Comment (adomasciplys) [reply server/moodle/mod/livequiz/classes/models/student_quiz_relation.php]: Yeah. I have no rational explanation. I have replaced it with a more reasonable comment

**Review by ChristofferGamel — APPROVED** [review]
:laughing:

**Review by DeeKahy — APPROVED** [review]

**Pull request merged**